contrib/database/sql/metrics.go

```diff
@@ -0,0 +1,54 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package sql // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/database/sql"
+
+import (
+	"database/sql"
+	"time"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/globalconfig"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+)
+
+const tracerPrefix = "datadog.tracer."
+
+// ref: https://pkg.go.dev/database/sql#DBStats
+const (
+	MaxOpenConnections = tracerPrefix + "sql.db.connections.max_open"
+	OpenConnections    = tracerPrefix + "sql.db.connections.open"
+	InUse              = tracerPrefix + "sql.db.connections.in_use"
+	Idle               = tracerPrefix + "sql.db.connections.idle"
+	WaitCount          = tracerPrefix + "sql.db.connections.waiting"
+	WaitDuration       = tracerPrefix + "sql.db.connections.wait_duration"
+	MaxIdleClosed      = tracerPrefix + "sql.db.connections.closed.max_idle_conns"
+	MaxIdleTimeClosed  = tracerPrefix + "sql.db.connections.closed.max_idle_time"
+	MaxLifetimeClosed  = tracerPrefix + "sql.db.connections.closed.max_lifetime"
+)
```

Comment:
NIT: Should these be public? (if not, we could still make them private before the next release starts on Monday, cc @dianashevchenko )

Reply:
No, there's no need for these to be public, that was just an oversight!